daemon/src/to_sse_event.rs

```diff
@@ -25,10 +25,26 @@ pub struct Cfd {
     pub profit_btc: Amount,
     pub profit_usd: Usd,
 
-    pub state: String,
+    pub state: CfdState,
     pub state_transition_timestamp: u64,
 }
 
+#[derive(Debug, Clone, Serialize)]
+pub struct CfdState {
+    pub label: String,
+    pub meta_state: CfdMetaSate,
+}
+
+#[derive(Debug, Clone, Serialize)]
+pub enum CfdMetaSate {
+    /// A CFD which has not been accepted or rejected
+    Open,
+    /// A CFD is which is on going
+    Running,
```

Comment:
I find this the `Open` vs. `Running` confusing. I think `Running` should be `Open`. 
For me `Open` means that your position is open, and in our context the position is open once we are on chain. Anything before is not `Open` but `ActiveOrders` (or `Opening` or `Setup` ...).

Reply:
I used the same terminology as LN markets for this but I'm happy to take another example as reference. 

On Bitmex they have: 

`Positions`: these are the current running positions
`Active Orders`: these are the orders which are not yet filled
`Order History`: passed positions
(I ignored the other states on Bitmex).

Reply:
I'm happy to just go with the same model as Bitmex when mapping the internal states to meta states for the UI.

Terminology wise: `Open` as we defined it means the Cfd is actually and `Open` position which in our context means it is on chain. I would still keep that internally in the daemon.

Reply:
How about these concrete enum variants:

`Opening`: has not been accepted or rejected yet
`Open`: has been accepted and is running (independent of transactions being included on-chain or not`
`Closed`: has been rejected or it's over

Reply:
> How about these concrete enum variants:
> 
> `Opening`: has not been accepted or rejected yet
> `Open`: has been accepted and is running (independent of transactions being included on-chain or not
> `Closed`: has been rejected or it's over

I don't think `Closed` is enough, another suggestion:

`Opening`: has not been accepted or rejected yet
`Open`: has been accepted and is running (independent of transactions being included on-chain or not
`Closed`: position that was once open and then was closed (currently maps final state `Refunded` only, other final states are still missing)
`Other`: `Rejected`, `SafelyAborted` (see #160)

Feel free to chose a different name than `Other` - e.g. `Aborted`, `Failed` (...).

Reply:
Why would you not combine `Rejected`/`SafelyAborted` with `Closed`?

Reply:
just throwing in another suggestion, perhaps a mid-way one: how about `Closed { Settled, Rejected, SafelyAborted}` - this would keep the main state from ballooning up, and at the same time allow us to keep the information how it ended?

Reply:
> Why would you not combine `Rejected`/`SafelyAborted` with `Closed`?

My reasoning woudl have been, that the user will likely not care about `SafelyAborted` / `Rejected` at all because those never made it on chain - no actual money was involved.  
But the user *might* want to see all the `Closed` ones (as in, he closed the position) because that determines wins/losses. That's why I would separate. Bitmex does not have `Rejected`/`SafelyAborted` because that cannot happen there I think :D

Reply:
> just throwing in another suggestion, perhaps a mid-way one: how about `Closed { Settled, Rejected, SafelyAborted}` - this would keep the main state from ballooning up, and at the same time allow us to keep the information how it ended?

This purely concerns how to group in the UI and not the internal state.